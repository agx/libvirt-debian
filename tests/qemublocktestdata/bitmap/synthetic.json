[
    {
        "iops_rd": 0,
        "detect_zeroes": "off",
        "image": {
            "virtual-size": 10485760,
            "filename": "/tmp/pull4.qcow2",
            "cluster-size": 65536,
            "format": "qcow2",
            "actual-size": 200704,
            "format-specific": {
                "type": "qcow2",
                "data": {
                    "compat": "1.1",
                    "lazy-refcounts": false,
                    "refcount-bits": 16,
                    "corrupt": false
                }
            },
            "dirty-flag": false
        },
        "iops_wr": 0,
        "ro": false,
        "node-name": "libvirt-1-format",
        "backing_file_depth": 0,
        "drv": "qcow2",
        "iops": 0,
        "bps_wr": 0,
        "write_threshold": 0,
        "dirty-bitmaps": [
            {
                "name": "current",
                "recording": true,
                "persistent": true,
                "busy": false,
                "status": "active",
                "granularity": 65536,
                "count": 0
            },
            {
                "name": "d",
                "recording": false,
                "persistent": false,
                "busy": false,
                "status": "disabled",
                "granularity": 65536,
                "count": 0
            },
            {
                "name": "c",
                "recording": false,
                "persistent": true,
                "busy": false,
                "status": "disabled",
                "granularity": 1234,
                "count": 0
            },
            {
                "name": "b",
                "recording": false,
                "persistent": true,
                "busy": true,
                "status": "disabled",
                "granularity": 65536,
                "count": 21314
            },
            {
                "name": "a",
                "recording": false,
                "persistent": true,
                "busy": false,
                "status": "disabled",
                "inconsistent": true,
                "granularity": 65536,
                "count": 0
            }
        ],
        "encrypted": false,
        "bps": 0,
        "bps_rd": 0,
        "cache": {
            "no-flush": false,
            "direct": false,
            "writeback": true
        },
        "file": "/tmp/pull4.qcow2",
        "encryption_key_missing": false
    },
    {
        "iops_rd": 0,
        "detect_zeroes": "off",
        "image": {
            "virtual-size": 197120,
            "filename": "/tmp/pull4.qcow2",
            "format": "file",
            "actual-size": 200704,
            "dirty-flag": false
        },
        "iops_wr": 0,
        "ro": false,
        "node-name": "libvirt-1-storage",
        "backing_file_depth": 0,
        "drv": "file",
        "iops": 0,
        "bps_wr": 0,
        "write_threshold": 0,
        "encrypted": false,
        "bps": 0,
        "bps_rd": 0,
        "cache": {
            "no-flush": false,
            "direct": false,
            "writeback": true
        },
        "file": "/tmp/pull4.qcow2",
        "encryption_key_missing": false
    }
]
