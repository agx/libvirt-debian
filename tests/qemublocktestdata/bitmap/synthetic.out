libvirt-1-format:
  current: record:1 busy:0 persist:1 inconsist:0 gran:65536 dirty:0
        d: record:0 busy:0 persist:0 inconsist:0 gran:65536 dirty:0
        c: record:0 busy:0 persist:1 inconsist:0 gran:1234 dirty:0
        b: record:0 busy:1 persist:1 inconsist:0 gran:65536 dirty:21314
        a: record:0 busy:0 persist:1 inconsist:1 gran:65536 dirty:0
